### DISCUSSION TITLE
How do I export the relationship fields of table and the pdf files using the Export feature

### DISCUSSION THREAD

**Discussion by dk13913** [Budibase/budibase#11563] [Help] [answered]
Selected all fields of the table to be downloaded including relationship fields but relationship fields are not at all coming in downloaded csv. Also, it does not download the pdf files uploaded in attachment field.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dk13913,

Relationships are not included in the CSV export. Should you decide to re-import the CSV, you will need to manually re-establish the relationships. This design choice stems from the fact that relationships can be deleted post-export. As a result, their existence cannot be guaranteed.

If you look at the CSV, you will see that it will include something like; `"[{'size':561208,'name':'pexels-pixabay-60597.jpg','extension':'jpg','key':'app_23423/attachments/123213-e44d-427a-a18f-f0ffff1fe078.jpg','url':'/files/signed/prod-budi-app-assets/app_235431/attachments/13523-e44d-427a-a18f-f0ffff1fe078.jpg?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=2432dsa%2F20230821%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230821T075022Z&X-Amz-Expires=3600&X-Amz-Signature=12321adasvac&X-Amz-SignedHeaders=host'}]"`. Exporting to CSV doesn't provide the downloadable files it just outputs the rows information, thats needed to re-link the uploaded attachments back to the row whenever importing it again. If you wish to download the file you will need to do this manually within Budibase.
